CMakeLists.txt

```diff
@@ -117,13 +124,19 @@ target_include_directories(xrealAirLinuxDriver
 		${OPENSSL_INCLUDE_DIRS}
 )
 
+if(DEFINED SYSTEM_INSTALL)
+  set(CURL_LIB ${CURL_LIBRARIES})
+else()
+  set(CURL_LIB libcurl)
+endif()
+
 target_link_libraries(xrealAirLinuxDriver
 		PRIVATE ${LIBEVDEV_LIBRARIES}
 		        ${JSONC_LIBRARIES}
 		        nrealAirLibrary
 		        Threads::Threads
 		        m
-		        libcurl
+            ${CURL_LIB}
```

Comment:
Inconsistent indentation

Reply:
BTW, prior to this project I wasn't a C developer and I don't know good formatting practices. If there's a good standard formatting that new devs to the project could easily adapt to in their IDE/whatever and there are good linting options for it, I'd be happy to adopt something like that. I've just been flying by the seat of my pants regarding formatting, honestly. No need to incorporate into this PR, we can talk offline if you have opinions here.

Reply:
Strange, locally it looks correctly indented. Maybe we are using different types (tabs, tab-length or whitespace).

I'm no c-dev myself. The project I worked on were c++ and then I used meson as build tool. But as far as I have seen, everyone does whatever he wants regarding formatting in c. There is no rule.

Reply:
You should be able to just copy-paste the whitespace leading up to my other libraries included here.